packages/blade/docs/guides/Installation.stories.mdx

```diff
@@ -40,125 +40,114 @@ Before you install the package, make sure that you have performed the following
 
 ## 🔜 Add blade libraries to your Figma project
 
 Coming Soon
 
-## 🔡 Install Fonts
+## 🔡 Installing Fonts
 
-We are using `Lato` font family which is not a system font so we need to perform some additional steps to download, install and configure it.
+We use 2 fonts. [TASA Orbiter](https://tasatype.localremote.co/) (for our headings), and [Inter](https://rsms.me/inter/) (for other text elements).
 
 ### Web Projects
 
-For web projects, we can either use Google fonts or self-hosted fonts. Self-hosted fonts have certain benefits that you can find [here](https://fontsource.org/docs/getting-started/introduction)
-
-#### Self-Hosted (Recommended)
-
-We recommend using self-hosted fonts. We'll use [Fontsource](https://fontsource.org/) which provides fonts as npm packages.
-
-- Install `Lato` font
+- To add these fonts to your project, you can import them from blade package in your root entry file.
 
-  ```shell
-  yarn add @fontsource/lato
+  ```ts
+  // Somewhere in root index.ts or entryBrowser.tsx file
+  import '@razorpay/blade/fonts.css';
   ```
 
-- Import within your app entry file(eg: `App.tsx`, `entryBrowser.tsx` etc.).
-  ```js
-  import '@fontsource/lato/400.css';
-  import '@fontsource/lato/400-italic.css';
-  import '@fontsource/lato/700.css';
-  import '@fontsource/lato/700-italic.css';
-  ```
+- While Blade handles setting font-family for its own components, you can set font-family globally as well in your global styles for any custom components or exceptions
 
-#### Google Fonts
+  ```ts
+  import { createGlobalStyle } from 'styled-components';
 
-There are downsides to using Google fonts which can be found [here](https://fontsource.org/docs/getting-started/introduction) but still for some reason if you have some use case where Google fonts makes more sense then you can [click this link](https://fonts.google.com/share?selection.family=Lato:ital,wght@0,400;0,700;1,400;1,700) and follow the instructions there to configure `Lato` for your applications.
+  const GlobalStyles = createGlobalStyle`
+  * { 
+    box-sizing: border-box;
+  }
+  
+  body {
+    margin: 0;
+    padding: 0;
+    font-family: ${(props) => props.theme.typography.fonts.family.text}
+  }
+  
+  h1, h2, h3, h4, h5, h6 {
+    font-family: ${(props) => props.theme.typography.fonts.family.heading};
+  }
+  `;
+  ```
 
 ### React Native Projects
 
-- Download `Lato` font from [here](https://fonts.google.com/share?selection.family=Lato:ital,wght@0,400;0,700;1,400;1,700)
-- Copy the font files and paste them under the directory `<project_root>/public/fonts`(**create the directory if it doesn't exist**)
-- Create a React Native config file at the root of your project - `<project_root>/react-native.config.js` and add the following content to it
+- Download fonts from <a href="https://github.com/razorpay/blade/blob/rebranding/typography-documentation/packages/blade/fonts/blade-fonts-react-native.zip">blade-fonts-react-native.zip</a> and unzip.
```

Comment:
Change this link to point to the master branch when merging.

Reply:
oh good catch. Yup I should use master only here